apps/web/src/pages/ApplicantDashboardPage/components/ReviewApplicationDialog.tsx

```diff
@@ -172,7 +170,24 @@ const ReviewApplicationDialog = ({
   return (
     <Dialog.Root open={isOpen} onOpenChange={setIsOpen}>
       <Dialog.Trigger asChild>
-        <PreviewList.Button label={poolName} />
+        <PreviewList.Button
+          label={
+            pool.name?.localized
+              ? intl.formatMessage(
+                  {
+                    defaultMessage:
+                      "<hidden>Application for </hidden>{poolName}",
+                    id: "LC1Rsg",
+                    description:
+                      "Text before application pool name in application preview list.",
+                  },
+                  {
+                    poolName: pool.name.localized,
+                  },
+                )
+              : nullMessage
+          }
+        />
```

Comment:
One for the polish ticket. Should we do the fallback in the argument so the button label would be "Application for not found" instead?